README.md

```diff
@@ -8,7 +8,7 @@
 
 [Join our slack!](https://join.slack.com/t/find-a-doc/shared_invite/zt-s4744a6o-MGaGHzLN5wB9aXeha3vdsQ)
 
-## Working with the submodule
+## Working with the submodule (will be deprecated after #195)
```

Comment:
Should we just take this out alltogether?

Reply:
Yes, but not until all the open PRs are merged. We need to get everyone synched on this